libs/website/ui-home/src/lib/introduction.tsx

```diff
@@ -78,6 +78,15 @@ export function Introduction() {
                   Bazel (by Google)
                 </a>
                 , &nbsp;
+                <a
+                  href="https://earthly.dev/?utm_source=monorepo.tools"
+                  rel="noreferrer"
+                  target="_blank"
+                  className="border-b border-yellow-500 transition hover:rounded hover:bg-yellow-500 hover:text-gray-800"
+                >
+                  Earthly (by Earthly Technologies, Inc)
```

Comment:
Simpler looks nicer I think.

```suggestion
                  Earthly (by Earthly Technologies)
```